pages/blog/august-2023.md

```diff
@@ -0,0 +1,53 @@
+
+## AACoT'23 London
+Speakers for AACoT'23 London Edition have been selected and informed; we are currently working on the schedule for the day. We anticipate a jam-packed day of inspiring talks and networking with esteemed industry experts. If you know any communities in London or ways we can maximize the AACoT'23 London event happening on the 20th of September, please do reach out to [Thulie](https://asyncapi.slack.com/team/U03CNJPMT7C) or [Lukasz](https://asyncapi.slack.com/team/UD698Q5LM) via Slack.
+
+## Financial Contributions
+We received new financial contributions this month. Please welcome our:
+- new Silver sponsor, [Svix](https://www.svix.com/), who will contribute $500 monthly.
+- new Bronze sponsor, [Rviewer](https://rviewer.io/), who will contribute $100 monthly.
+
+Huge thank you to our one-time sponsor, [Simlot](https://www.simplot.com/), for donating $944.
+
+## AsyncAPI Bounty Program Trial
+We're thrilled to share that the [AsyncAPI Bounty Program Trial](https://github.com/orgs/asyncapi/discussions/541) has been completed. We started the trial with a total of 7 projects consisting of coding, design, and technical writing tasks, categorized into medium and advanced levels. We had pre-set rules for the bounty trial, and everything went smoothly when it came to task assignment and completion. We also learned valuable insights during the trial, such as timeline extension and handling additional tasks necessary for task completion that will be implemented in the official program. Here are the accomplished tasks and the exceptional assignees:
+
+| Bounty Issue Link | Scope | Category | Type | Assignee | Status |
+-------------------- | ------- | ---------- | ----- | ---------- | ------- |
+| https://github.com/asyncapi/website/issues/843 | Enable caching on a website built on Netlify | Medium Level | Coding | [Sambhav Gupta](https://github.com/sambhavgupta0705) | Complete |
+| https://github.com/asyncapi/cli/issues/358 | Add a Dockerfile and release pipeline necessary to publish it | Medium Level | Coding | [Savio Dias](https://github.com/Savio629) | Complete | 
+| https://github.com/asyncapi/cli/issues/123 | Configure CLI commands docs generation into existing flows | Medium Level | Coding | [Mahfuza Humayra Mohona](https://github.com/mhmohona) | Complete 
+| https://github.com/asyncapi/cli/issues/38 | Enable CLI to read the context from CWD | Advanced Level | Coding | [Viacheslav Turovskyi](https://github.com/aeworxet) | Complete |
+| https://github.com/asyncapi/generator/issues/943 | Create a first tutorial for AsyncAPI Generator | Advanced Level | Doc's | [Florence Njeri](https://github.com/Florence-Njeri) | Complete |
+| https://github.com/asyncapi/community/issues/672 | Create an infographic with information about AsyncAPI Initiative finance | Advanced Level | Design | [AISHAT MUIBUDEEN](https://github.com/Mayaleeeee) | Complete |
```

Comment:
look at you gooo @Mayaleeeee, remember when I first met you via GSoD interviews and we realized your passion was contributing to design?!?! ✨✨✨

Reply:
> look at you gooo @Mayaleeeee, remember when I first met you via GSoD interviews and we realized your passion was contributing to design?!?! ✨✨✨

Of course I do! 😂😂